contracts/contracts/PoolDeposits.sol

```diff
@@ -204,11 +209,12 @@ contract PoolDeposits {
   /// @param amount the user wants to deposit into the DAOcare pool
   function deposit(uint256 amount)
     external
-    blankUser
+    hasNotEmergancyVoted
```

Comment:
It is important that they have not emergency voted. Only allow `exit` to be called after they have voted for an emergancy - see comment on that function. 

No reason for user to do a partial withdrawal if they believe there is an `emergency`.